packages/api/cms-api/src/user-permissions/user-permissions.module.ts

```diff
@@ -22,8 +27,18 @@ import { UserPermissionsAsyncOptions, UserPermissionsOptions, UserPermissionsOpt
             provide: USER_PERMISSIONS_SERVICE,
             useClass: UserPermissionsService,
         },
+        ContentScopeService,
+        {
+            provide: CONTENT_SCOPE_SERVICE,
+            useClass: ContentScopeService,
+        },
```

Comment:
Why use a constant here, and not the shorthand for services?

Reply:
c70ba8f4087be80e4fd537ab86a737918bd1abfd